client/setdownedstate.lua

```diff
@@ -111,22 +111,27 @@ local function playLastStandAnimation(ped)
     end
 end
 
+---Player is able to send a notification to EMS there are any on duty
+local function handleRequestingEms()
+    if not isEmsOnDuty() then return end
```

Comment:
Calling a callback each frame is very bad

Reply:
Cacheing with 1 minute eviction instead now.